src/azure-cli/azure/cli/command_modules/sqlvm/_help.py

```diff
@@ -67,19 +67,22 @@
         - name: Create a SQL virtual machine with specific sku type and license type.
           text: >
             az sql vm create -n sqlvm -g myresourcegroup -l eastus --image-sku Enterprise --license-type AHUB
+        - name: Create a SQL virtual machine with NoAgent type, only valid for EOS SQL 2008 and SQL 2008 R2.
+          text: >
+            az sql vm create -n sqlvm -g myresourcegroup -l eastus --license-type AHUB --sql-mgmt-type NoAgent --image-sku Enterprise --image-offer SQL2008-WS2008R2
         - name: Enable R services in SQL2016 onwards.
           text: >
-            az sql vm create -n sqlvm -g myresourcegroup -l eastus --license-type PAYG --enable-r-services true
+            az sql vm create -n sqlvm -g myresourcegroup -l eastus --license-type PAYG --sql-mgmt-type Full --enable-r-services true
```

Comment:
>--sql-mgmt-type Full - [](start = 87, length = 22)

I am thinking to change those text as LightWeight as well, since by default for create we are taking LightWeight

Reply:
This are just specific create options if customer wants to create + enable sql manageability. Do you think for the basic commands customer needs the LightWeight default set?